guide/blueprints/_elect-primary-policies.md

```diff
@@ -0,0 +1,149 @@
+
+There are a collection of policies, enrichers, and effectors to assist with common
+failover scenarios and more general anything which wants a primary elected and
```

Comment:
general -> generally,

Reply:
which wants a primary elected and re-elected. -> which requires the election and re-election of a primary member.